cluster-capi-kind/docker_machine_template_control_plane.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: DockerMachineTemplate
+metadata:
+  name: nephio-control-plane
+  namespace: default
+spec:
+  template:
+    spec:
+      extraMounts:
+      - containerPath: /var/run/docker.sock
```

Comment:
Wondering if mutator pipeline can be used to replace this with podman related values if podman support is desired

Reply:
Yeah, that will be an interesting test case for how we might do that!